docker-compose-with-images.yaml

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+

```

Comment:
Add a comment explaining the purpose of this Docker Compose configuration, similar to other Docker Compose files.